src/WidgetBase.ts

```diff
@@ -433,6 +440,7 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 	}
 
 	public __render__(): VirtualDomNode | VirtualDomNode[] {
+		this._nodeHandler.clear();
```

Comment:
is this the correct place for this? so it clears regardless of whether the widget was invalidated.... I don't think that this is the equivalent to when it was executed in the `@beforeRender`...